DSLink.UWP/UWPPlatform.cs

```diff
@@ -0,0 +1,15 @@
+using DSLink.Connection;
+using PCLStorage;
+
+namespace DSLink.UWP
+{
+    public class UWPPlatform
+    {
+        public static void Initialize()
+        {
+            Configuration.StorageBaseFolder = FileSystem.Current.LocalStorage;
+            Websockets.Universal.WebsocketConnection.Link();
```

Comment:
Or maybe it's all right, I just never use "Link" as a verb, because well, I'm not natively speaking english.